stable/gcloud-sqlproxy/templates/_helpers.tpl

```diff
@@ -11,6 +11,6 @@ Create a default fully qualified app name.
 We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
 */}}
 {{- define "gcloud-sqlproxy.fullname" -}}
-{{- $name := default .Chart.Name .Values.nameOverride -}}
-{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- $name := .Values.nameOverride | default (printf "%s-%s" .Release.Name .Release.Name) -}}
```

Comment:
`nameOverride` is meant for the `name` template. Either add a `fullnameOverride` or do something like this: https://github.com/kubernetes/charts/blob/master/incubator/oauth-proxy/templates/_helpers.tpl#L14-L18